resources/group.go

```diff
@@ -225,6 +224,27 @@ func (obj *GroupRes) CheckApply(apply bool) (checkOK bool, err error) {
 type GroupUID struct {
 	BaseUID
 	name string
+	gid  *uint32
+}
+
+// IFF aka if and only if they are equivalent, return true. If not, false.
+func (obj *GroupUID) IFF(uid ResUID) bool {
+	res, ok := uid.(*GroupUID)
+	if !ok {
+		return false
+	}
+	eq := true
+	if obj.gid != nil && res.gid != nil {
+		if *obj.gid != *res.gid {
+			eq = false
```

Comment:
if we know this is false, why not just return false` now and avoid the eq var?
(same point below and at end)